File: flowcontrolls/Looping/prime-range.py
#print prime numbers in a range

#n1=int(input("Enter range1:"))
#n2=int(input("Enter range2:"))
#flag=0
#num=0
#for num in range(n1,(n2+1)):
#    for i in range(2,num):
#        if(num%i==0):
#            flag=flag+1
#            break
#        else:
#            flag=0
#    if(flag==0):
#        print(num)

lower=int(input("Enter lower limit:"))#5
upper=int(input("Enter upper limit:"))#17
flag=0
for num in range(lower,upper+1):
    for i in range(2,num):
        if(num%i==0):
            flag=flag+1
            break
        else:
            flag=0
    if(flag==0):
        print(num)

File: Html_sections/htmlsections/luproject/luminar-project/Javascript/Looping_statements/primerange.js
var lower=2;
var upper=10;
var flag=0;
for(let i=lower;i<=upper;i++){
    for(let j=2;j<i;j++){
        if(i%j==0){
            flag=flag+1;
            break;
        }
        else{
            flag=0;
        }

    }
    if(flag==0){
        console.log(i);
    }
}
